### PR TITLE
pytest

### PR DESCRIPTION
**Pull request by durandom**
minimal pytest setup

### CODE REVIEW

**Comment by durandom** [comment]
@goern @fridex pls review

File: Pipfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 rx = "*"
 pandas = "*"
+pytest = "*"
```
Comment (fridex) [inline Pipfile]: I would suggest moving pytest to dev-packages by:

```console
pipenv install --dev pytest
```

Comment (durandom) [reply Pipfile]: thanks

**Comment by durandom** [comment]
fixes #22

**Pull request merged**